test/arrays/vass/const-checking.aliases.good

```diff
@@ -1,9 +1,2 @@
-const-checking.aliases.chpl:8: In function 'main':
-const-checking.aliases.chpl:10: error: illegal lvalue in assignment
-const-checking.aliases.chpl:12: error: illegal lvalue in assignment
-const-checking.aliases.chpl:14: error: creating a non-const alias 'aa2' of a const array or domain
-const-checking.aliases.chpl:18: error: creating a non-const alias 'dd2' of a const array or domain
-const-checking.aliases.chpl:23: error: illegal lvalue in assignment
-const-checking.aliases.chpl:24: error: illegal lvalue in assignment
-const-checking.aliases.chpl:27: error: illegal lvalue in assignment
-const-checking.aliases.chpl:28: error: illegal lvalue in assignment
```

Comment:
This is another one where it seems like ensuring that these error messages still occur is important.  Originally, I created a clone of this test that commented out lines 14 and 18 on the hypothesis that different passes created the errors and that I might still get the errors on the original lines once I got past the first few.  But I can't remember what happened in that experiment (only that I've been tripping over a "missing .good" error message ever since...).  Probably I timed out before completing it...  If creating such a variation of the test did not generate warnings for the other lines, that seems like another case to file a future against?